lib/models/applicant.rb

```diff
@@ -23,4 +26,11 @@ def prefers_interest
     !prefers_nearby
   end
   alias_method :prefers_interest?, :prefers_interest
+
+  private 
+
+    def compute_grid_id
+      @grid = Grid.intersecting_grid(location: location)
+      self.grid_id = @grid.g250m_id if @grid
```

Comment:
Since this is a hook method and not a controller, we can change the `@grid` instance variable to a local variable.